slam-libraries/viam-cartographer/src/mapping/map_builder.cc

```diff
@@ -96,6 +96,20 @@ cartographer::sensor::TimedPointCloudData MapBuilder::GetDataFromFile(
 
 // TODO: There might still be a lot of room to improve accuracy & speed.
 // Might be worth investigating in the future.
+cartographer::transform::Rigid3d MapBuilder::GetLastGlobalPose(
+    int trajectory_id) {
+    if (GetLocalSlamResultPoses().size() > 0) {
+        auto latest_local_pose_ = GetLocalSlamResultPoses().back();
```

Comment:
[nit] Rename `latest_local_pose_` to `latest_local_pose`. The trailing `_` is just an artifact